src/backend/gdb_expansion.ts

```diff
@@ -223,7 +224,7 @@ export function expandValue(variableCreate: Function, value: string, root: strin
 				val = "Object@" + val;
 			}
 		} else if (typeof val == "string" && val.startsWith("@0x")) {
-			ref = variableCreate(getNamespace("*&" + name.substr));
+			ref = variableCreate(getNamespace("*&" + name.substring(1)));
```

Comment:
ah nice catch, do you know why you remove the first character here? It looks like maybe just passing `name` instead might be correct instead. Sadly there is no test for this to actually verify this.

Reply:
That was just a guess. Feel free to change it.